### DISCUSSION TITLE
Does coque tts has API?

### DISCUSSION THREAD

**Discussion by kaz1ngton** [coqui-ai/TTS#1634] [General Q&A] [answered]
Im wondering if coque tts has Rest API?

**Comment by erogol** [comment, marked as answer]
you can try ```tts-server```. It is not a rest API but the closest thing in 🐸TTS.

**Reply by thorstenMueller** [reply]
Just a short addition @kaz1ngton . When your `tts-server` is up and running you can just call `/api/tts?text=test` as a `GET` request.
If you want to embed it in a Python app you could check https://pypi.org/project/cTTS/.

**Reply by luochen1990** [reply]
Is the server compatible with OpenAI API ?

**Reply by K4leri** [reply]
> Just a short addition @kaz1ngton . When your `tts-server` is up and running you can just call `/api/tts?text=test` as a `GET` request. If you want to embed it in a Python app you could check https://pypi.org/project/cTTS/.

I spent two hours trying to find the api, but still can't figure out what fields are available for input. I need to enter the model name, voice selection, but the server keeps sending 500 errors. 

I am downloading docker version
![изображение](https://github.com/user-attachments/assets/3bdf32c5-576e-427f-8d09-bad07d12e328)

but then it says me that it is nor production ready. What is for in such way ? why it contains in official docs ?

```cmd
INFO:werkzeug:WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (::)
 * Running on http://[::1]:5002
 * Running on http://[::1]:5002
INFO:werkzeug:Press CTRL+C to quit
```

**Reply by luciaokay** [reply]
@K4leri have you found a solution eventually?

**Reply by eginhard** [reply]
@K4leri @luciaokay I added some documentation on the available parameters: https://coqui-tts.readthedocs.io/en/latest/server.html#parameters

I'd also recommend to use [our fork](https://github.com/idiap/coqui-ai-TTS) (available via `pip install coqui-tts`). This repo is not maintained anymore and the server here doesn't support all Coqui models.
